File: js_katas/jasmine-standalone-1.0.0/spec/roman_spec.js
describe("RomanNumeral", function () {
  var numerals = {
    1: 'I',
    2: 'II',
    4: 'IV',
    5: 'V',
    10: 'X',
    20: 'XX',
    25: 'XXV',
    29: 'XXIX',
    30: 'XXX',
    40: 'XL',
    45: 'XLV',
    50: 'L',
    54: 'LIV',
    90: 'XC',
    99: 'XCIX',
    100: 'C',
    300: 'CCC'
  }

  _.each(numerals, function(roman_num, natural_num){
    it('returns' + roman_num + 'for' + natural_num, function(){
      expect(roman(natural_num)).toEqual(roman_num)
    })
  })
})

File: ruby_katas/roman_spec.rb
require_relative './roman'

describe Roman do 
[
  [1, 'I'],
  [2, 'II'],
  [3, 'III'],
  [4, 'IV'],
  [5, 'V'],
  [6, 'VI'],
  [7, 'VII'],
  [9, 'IX'],
  [10, 'X'],
  [11, 'XI'],
  [12, 'XII'],
  [19, 'XIX'],
  [20, 'XX'],
  [21, 'XXI'],
  [25, 'XXV'],
  [29, 'XXIX'],
  [30, 'XXX'],
  [40, 'XL']

  ].each do | natural_number, roman_numeral |
    it "converts #{natural_number} to #{roman_numeral}" do 
      Roman.of(natural_number).should == roman_numeral
    end
  end
end